clang/lib/AST/ASTImporter.cpp

```diff
@@ -4437,6 +4441,31 @@ ExpectedDecl ASTNodeImporter::VisitFriendDecl(FriendDecl *D) {
   return FrD;
 }
 
+ExpectedDecl ASTNodeImporter::VisitFriendPackDecl(FriendPackDecl *D) {
+  // Import the major distinguishing characteristics of a declaration.
+  DeclContext *DC, *LexicalDC;
+  if (Error Err = ImportDeclContext(D, DC, LexicalDC))
+    return std::move(Err);
+
+  auto ToInstantiatedFromFriendOrErr =
+      Importer.Import(D->getInstantiatedFromFriendDecl());
```

Comment:
nit: It would be great to spell out the type name if it is not too long - I'm also unfamiliar with the ASTImporter, though :)

Reply:
Hmm, I basically just copy-pasted stuff from the functions that handle `UsingPackDecl`s and `FriendDecl`s, and it seems that those always use `auto`, so this would probably be better left for a separate NFC patch imo, but I suppose I could change it in this function here.